libultraship/libultraship/ResourceMgr.cpp

```diff
@@ -19,7 +20,19 @@ namespace Ship {
 		}
 	}
 
-	ResourceMgr::~ResourceMgr() {
+    ResourceMgr::ResourceMgr(std::shared_ptr<Window> Context, const std::vector<std::string> &OTRFiles, const std::string &PatchesPath, const std::unordered_set<uint32_t> &ValidHashes)
+        : Context(Context), bIsRunning(false), FileLoadThread(nullptr)
+    {
+        OTR = std::make_shared<Archive>(OTRFiles, PatchesPath, ValidHashes, false);
+
+        gameVersion = OOT_UNKNOWN;
```

Comment:
Should probably add a new unknown enum value that doesn't imply OOT at all.

Reply:
I don't think I actually changed that line (though my editor may have subtly changed some formatting at some point), and the macros for the various OOT hashes were already in LUS and probably shouldn't be, but I'll rename this one to just UNKNOWN for now.

Reply:
Done.